src/pure/target.mli

```diff
@@ -101,12 +100,12 @@ module Condition : sig
   (**
     An execution anti-condition; the condition defines when a target is
     ready and therefore should be run if the condition is {emph not} met: {ul
-    {li with [`Never] the target always runs (because never “ready”),}
-    {li with [`Satisfied] the target never runs (a bit useless),}
+    {li with [`Never] the target always runs and then fails
+                      (because never “ready”),}
     {li with [`Volume_exists v] the target runs if the volume does not exist
     ([make]-like behavior).}
     {li with [`Volume_size_bigger_than (v, sz)] Ketrew will get the total size
-    of the volume (in bytes) and check that it is bigger.}
+    of the volume (in bytes) and check that it is {b bigger-or-equal}.}
```

Comment:
I think for now what we have is fine.

In the future, this can be implemented:

```ocaml
  type t = [
    | `Never
    | `Volume of Volume.t * [
        | `Exists
        | `Has_size of [ `LT | `GT | `GE | `LE | `EQ | `NEQ ] * int
        ] list
    | `Command of Command.t * [
          | `Returns of int
          | `Outputs of [`Stdout of string_predicate | `Stderr of string_predicate ]
        ] list
    | `Or of t list
    | `And of t list
  ]
```

And we will be able to keep backwards compatible cases if we want.